.cargo/config

```diff
@@ -0,0 +1,2 @@
+[build]
+target-dir= "../target"
```

Comment:
How about the top-level crate? Will it use the target directory in the parent directory?

Reply:
all build cache will store on `../target`